zuul.d/edpm_periodic.yaml

```diff
@@ -11,6 +11,18 @@
     parent: cifmw-crc-podified-edpm-baremetal
     vars: *edpm_vars
 
+- job:
+    name: periodic-edpm-build-push-images-centos-9-master
+    parent: cifmw-base-edpm-build-images
+    vars:
+      buildah_login: true
+      cifmw_repo_setup_branch: master
+      cifmw_repo_setup_promotion: podified-ci-testing
+      openstack_release: master
```

Comment:
I am not sure where this is used.

Reply:
I tried to remove this earlier but job fail in pre earlier because of removal: https://logserver.rdoproject.org/47/347/c9a0f7ce58f318a393826c440eb1f26348611dde/github-check/periodic-edpm-build-push-images-centos-9-antelope/d01b9e8/job-output.txt

we need this because of this condition here: https://review.rdoproject.org/r/plugins/gitiles/config/+/refs/heads/master/playbooks/crc/podman.yaml#31

Reply:
Thank you for adding the links, make sense!